lib/utils.js

```diff
@@ -67,7 +67,7 @@ const cleanSelectors = (str) => str
   .filter(selector => selector);
 
 
-const renderOperators = ['>>', '->'];
+const renderOperators = ['>>', '->', '<>'];
```

Comment:
@shanebo I was not able to fully figure out the purpose of `getPaneSelectors`, which uses `renderOperators`. I think this should be added here, but I am not 100% certain.